pkg/proxy/service_test.go

```diff
@@ -17,11 +17,11 @@ limitations under the License.
 package proxy
 
 import (
+	"github.com/davecgh/go-spew/spew"
```

Comment:
Group this import separately in the bottom

Reply:
Done. (must have been a "gofmt" thing)